RBS.Auth.WebApi/RBS.Auth.WebApi/Models/LoginRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace RBS.Auth.WebApi.Models;
+
+public class LoginRequest
+{
+    [Required] 
+    [EmailAddress] 
+    public string Email { get; set; }
```

Comment:
Consider to use `FluentValidator` library, it is more flexible and doesn't overload the model

Reply:
Fixed